lib/driver.rb

```diff
@@ -0,0 +1,58 @@
+require 'csv'
+require 'awesome_print'
+
+require_relative 'csv_record'
+require 'pry'
+
+module RideShare
+  class Driver < CsvRecord
+    attr_reader :id, :name, :vin, :trips
+    attr_accessor :status
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: nil)
+      super(id)
+
+      @name = name
+      @vin = vin
+      @status = status
+      @trips = trips || []
+
+      unless @vin.length == 17 && (@vin.is_a? String)
+        raise ArgumentError.new ('Your VIN is wrong.')
+      end
+
+      approved_status = [:AVAILABLE, :UNAVAILABLE]
+
+      unless approved_status.include?(@status)
+        raise ArgumentError.new ('You must provide one of the following statuses :available, :unavailable')
```

Comment:
Symbols are case sensitive:

```suggestion
        raise ArgumentError.new ('You must provide one of the following statuses :AVAILABLE, :UNAVAILABLE')
```